ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -97,6 +104,25 @@ public static DeleteRecord deserializeDeleteRecord(InputStream stream) throws IO
     }
   }
 
+  public static ByteBuffer deserializeBlobEncryptionKey(InputStream stream) throws IOException, MessageFormatException {
```

Comment:
MessageFormatRecord has detailed javadoc for each version of each kind of record. It has conventionally avoided javadoc for individual methods, probably because these are self descriptive. Just followed the same pattern.